test/canvas.test.js

```diff
@@ -377,6 +384,103 @@ describe('Canvas', function () {
     });
   });
 
+  describe('#toBuffer("raw")', function() {
+    var canvas = new Canvas(10, 10)
+        , ctx = canvas.getContext('2d');
+
+    ctx.clearRect(0, 0, 10, 10);
+
+    ctx.fillStyle = 'rgba(200, 200, 200, 0.505)';
+    ctx.fillRect(0, 0, 5, 5);
+
+    ctx.fillStyle = 'red';
+    ctx.fillRect(5, 0, 5, 5);
+
+    ctx.fillStyle = '#00ff00';
+    ctx.fillRect(0, 5, 5, 5);
+
+    ctx.fillStyle = 'black';
+    ctx.fillRect(5, 5, 4, 5);
+
+    /** Output:
+     *    *****RRRRR
+     *    *****RRRRR
+     *    *****RRRRR
+     *    *****RRRRR
+     *    *****RRRRR
+     *    GGGGGBBBB-
+     *    GGGGGBBBB-
+     *    GGGGGBBBB-
+     *    GGGGGBBBB-
+     *    GGGGGBBBB-
+     */
+
+    var buf = canvas.toBuffer('raw');
+    var stride = canvas.stride;
+
+    var isBE = (function() {
```

Comment:
`os.endianness()` was introduced in Node v0.9.4, but Travis runs the tests on Node v0.8.